python_modules/dagster/dagster/_utils/caching_instance_queryer.py

```diff
@@ -586,11 +590,9 @@ def is_reconciled(
             asset_graph.get_partitions_def(asset_partition.asset_key),
             (TimeWindowPartitionsDefinition, DynamicPartitionsDefinition),
         )
-        for parent in asset_graph.get_parents_partitions(
-            self,
-            self._evaluation_time,
-            asset_partition.asset_key,
-            asset_partition.partition_key,
+
+        for parent in asset_graph.get_valid_parent_partitions(
+            dynamic_partitions_store, self._evaluation_time, asset_partition
```

Comment:
a quick nit before a more thorough review is that we can just pass in `self` here, rather than re-routing the caching instance queryer into itself at all the call sites.

Reply:
good call